bisection/application.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python2
+
+"""The fake-core binary of your application
+
+Assume this is the core binary of your application. Assume your goal
+is to keep this binary running without any problems. The purpose is
+to train you using `git bisection` to find out which commit causes
+this binary stopped to work.
+
+The algorithm of this script is to find out how many lines of
+comments are there in the repo. If it exceeds the number we pass
+to this program, it'll raise an exception.
+"""
+
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+COMMENT_STYLES = {
```

Comment:
Should there be a inclusion for XML as is that covered in the same context as HTML ?

Reply:
Thanks. Added the same comment style as html